### ISSUE TITLE
Review/replace term "aggregation definition" for composed rules

### ISSUE THREAD

**Issue by EmmaJP** [w3c/wcag-act#241]
Apparently there is some duplication of the word "aggregation" across ACT and Auto-WCAG work.

The new composed rules may need to replace "aggregation definition" with something else that references how a set of atomic rules work together for a composed ruled. Maybe "Composition Definition" or "Composed Definition".

https://github.com/auto-wcag/auto-wcag/blob/7d081504edfbde9cb48b13a9b5c7540b8ea1b553/pages/design/composed-template-empty.md

**Comment by moekraft** [comment]
From Shadi, In Rule Types the final section should be moved to Aggregation section. Any change to the Aggregate definition should take this into account as well.

 "A composed rule defines how the outcomes from its atomic rules are aggregated into a single outcome for each applicable test target. Note that atomic rules in a composed rule MAY have different applicability. Because of this, not every element applicable within the composed rule is tested by every atomic rule. Atomic rules MAY also be disabled during a test run due to accessibility support concerns. See Accessibility Support for details.
Composed rule: Header cells in HTML tables (WCAG 2 Success Criterion 1.3.1).
This rule uses results from the following atomic rules:
•	Header indicated through implicit scope 
•	Header indicated through the scope attribute 
•	Header indicated by using the headers attribute 
•	Header indicated by using ARIA labels 
If any one of these rules passes, the table cell can pass the composed rule." 

[Accessibility Conformance Testing.docx](https://github.com/w3c/wcag-act/files/2293715/Accessibility.Conformance.Testing.docx)

**Comment by annethyme** [comment]
Agree that "Aggregation Definition" is a bit confusing.
My suggestion would be to get rid of the "Test Definition" and "Aggregation Definition" headers all together. 

PR to illustrate how this could be done: https://github.com/w3c/wcag-act/pull/247

**Comment by cpandhi** [comment]
To remove duplication and confusion, i would suggest:
Section 9: Atomic rule definition
Section 10: 
Section 11: Composed rule definition (probably this should be section 10, followed by the list of rules that goes into the composed rules as a sub bullet or a section after.)

**Comment by moekraft** [comment]
Addressed by PR #247

**Comment by cpandhi** [comment]
Looks like the example 5 needs correction, it has  "Video elements have a media alternative" repeating as stated below. May be the 3rd option could be "Static Text Alternative" 
A composed composite rule for WCAG 2.1 Audio Description or Media Alternative aggregates applies a logic across results from three atomic rules. The expectation of the composed composite rule is as follows:

    For each test target, the outcome of one of the following rules is passed :

        Video elements have an audio description
        Video elements have a media alternative
        Video elements have a media alternative

**Comment by moekraft** [comment]
Merged PR #247
